react-native-libraries.json

```diff
@@ -10934,5 +10934,18 @@
       "https://github.com/RichardRNStudio/react-native-find-local-devices/blob/main/example"
     ],
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Netizen-Teknologi/avatar-placeholder",
+    "npmPkg": "@netizen-teknologi/avatar-placeholder",
+    "nameOverride": "GENERATE AVATAR PLACEHOLDER",
```

Comment:
Hello @rinaldycodes, thanks for the addition! 👍 

One nit before we can ship it, let's stick to npm package name:
```suggestion
```

Reply:
thank you @Simek